app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cmd, src_file, dst_file = command.split(" ")
+
+    if cmd != "cp" or src_file == dst_file:
+        return
+
+    with open(src_file, "r") as file_in, open(dst_file, "w") as file_out:
```

Comment:
Clean Code: You correctly open the source file for reading and the destination file for writing. However, you use 'file_in' and 'file_out' as variable names, which are not very descriptive. It would be better to use 'source_file' and 'destination_file' as variable names, as suggested in the checklist.